examples/super-todo/src/views/CustomerListView.tsx

```diff
@@ -115,58 +121,11 @@ const CustomerListView = () => {
             {/* Lists */}
             <ListTabPanel lists={lists} />
 
-            {/* Todos */}
-            <TabPanel>
-              <Box css={{ marginTop: "large" }}>
-                <Box css={{ marginTop: "small" }}>
-                  <Checkbox
-                    label={
-                      <>
-                        <Box>Respond to janefisher@gmail.com</Box>
-                        <Box css={{ color: "secondary", marginTop: "xxsmall" }}>
-                          @tayler
-                        </Box>
-                      </>
-                    }
-                  />
-                </Box>
-                <Box css={{ marginTop: "small" }}>
-                  <Divider/>
-                </Box>
-                <Box css={{ marginTop: "small" }}>
-                  <Checkbox
-                    label={
-                      <>
-                        <Box>Respond to ryan.lee@gmail.com</Box>
-                        <Box css={{ color: "secondary", marginTop: "xxsmall" }}>
-                          @tayler
-                        </Box>
-                      </>
-                    }
-                  />
-                </Box>
-                <Box css={{ marginTop: "small" }}>
-                  <Divider/>
-                </Box>
-                <Box css={{ marginTop: "small" }}>
-                  <Checkbox
-                    label={
-                      <>
-                        <Box>Respond to chris_p@hotmail.com</Box>
-                        <Box css={{ color: "secondary", marginTop: "xxsmall" }}>
-                          @tayler
-                        </Box>
-                      </>
-                    }
-                  />
-                </Box>
-                <Box css={{ marginTop: "small" }}>
-                  <Divider/>
-                </Box>
-              </Box>
-            </TabPanel>
+            {/* Render Todos */}
+            {showTodos(showCompletedTodos, setTodoType, showCompletedTodos)}
```

Comment:
Looks like showTodos only takes in the first two arguments

Reply:
Good catch. Done.